test/helpers/clinics_helper_test.rb

```diff
@@ -0,0 +1,5 @@
+require 'test_helper'
+
+class ClinicsHelperTest < ActionView::TestCase
```

Comment:
In terms of my time available probably best to roll as is or do what you are talking about. I have a friend in town so no time for the next couple days.